ts/screens/authentication/cie/ActivateNfcScreen.tsx

```diff
@@ -0,0 +1,97 @@
+import { ListItemInfo } from "@pagopa/io-app-design-system";
+import React, { useCallback } from "react";
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { Route, useNavigation, useRoute } from "@react-navigation/native";
+import { Alert } from "react-native";
+import I18n from "../../../i18n";
+import { useIOSelector } from "../../../store/hooks";
+import { isNfcEnabledSelector } from "../../../store/reducers/cie";
+import { IOStackNavigationProp } from "../../../navigation/params/AppParamsList";
+import { AuthenticationParamsList } from "../../../navigation/params/AuthenticationParamsList";
+import ROUTES from "../../../navigation/routes";
+import * as cieUtils from "../../../utils/cie";
+import ScreenWithListItems from "../../../components/screens/ScreenWithListItems";
+import { CieCardReaderScreenNavigationParams } from "./CieCardReaderScreen";
+
+const ActivateNfcScreen = () => {
+  const isEnabled = useIOSelector(isNfcEnabledSelector);
+  const isNfcEnabled = pot.getOrElse(isEnabled, false);
+  const navigation =
+    useNavigation<
+      IOStackNavigationProp<
+        AuthenticationParamsList,
+        typeof ROUTES.CIE_ACTIVATE_NFC_SCREEN
```

Comment:
Since `typeof ROUTES.CIE_ACTIVATE_NFC_SCREEN` is used twice, what do you think about moving it in a custom type defined out of component lifecycle? e.g. `type CieActivateNFCKey = typeof ROUTES.CIE_ACTIVATE_NFC_SCREEN`